lib/rpmsg/rpmsg.c

```diff
@@ -293,8 +286,7 @@ int rpmsg_create_ept(struct rpmsg_endpoint *ept, struct rpmsg_device *rdev,
 		 */
 	}
 
-	rpmsg_initialize_ept(ept, name, addr, dest, cb, unbind_cb);
-	rpmsg_register_endpoint(rdev, ept);
+	rpmsg_initialize_ept(ept, rdev, name, addr, dest, cb, unbind_cb);
```

Comment:
1. Avoid the code duplication
2. Avoid the wrong usage

Reply:
> 1. Avoid the code duplication

In this case, it would be more optimized to put `rpmsg_initialize_ept` in rpmsg.c and remove the `inline`, to let the compiler optimize the code.

> 2. Avoid the wrong usage
Still no clear for me, as usage of `rpmsg_register_endpoint` is internal only. 

Another point:
Use  `rpmsg_register_endpoint`/`rpmsg_unregister_endpoint` as internal interface, seems more readable than `rpmsg_initialize_ept`/`rpmsg_unregister_endpoint`.

Reply:
Done.